### ISSUE TITLE
FGA on OnePlus 9 pro

### ISSUE THREAD

**Issue by saif-k** [Fate-Grand-Automata/FGA#1246]
### Preparation

- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)
- [X] My question isn't answered in any other issues (even the closed ones)

### Your question

So I'm trying to run FGA on my oneplus 9 pro every time I run it on default settings it doesn't do the correct skill for instance I run musashi and 2 castorias the commands are as follows dge1h1f1i1b4,#,4,#,4.
First skill is obviously castorias first skill but it clicks musashis 3rd skill instead and then messes up the entire setup.
I have tried other options in the setting like ignore notch calculation, reducing the similarities. But it doesn't do anything so I played around with game area mode and well it made it run when I setup as 
Left: 130 right: 100 top: 10 bottom: 10 the run works fine issue is now friend selection and refill, reducing Min similarities to 50 it picks a random friend and it continues but no refill.
Thats My situation if anyone has a solution please help.
Fyi I have set both fgo and fga to full screen.

**Comment by reconman** [comment]
I need a video with debug mode.

**Comment by saif-k** [comment]
Okay attached tried to upload direct said file is to big so here's a Dropbox link
https://www.dropbox.com/s/ggp4n03678ujvyy/record.mp4?dl=0

**Comment by saif-k** [comment]
![Uploading Screenshot_2022-07-13-21-31-03-77_577bcc70c040536a6d306ef99c704547.jpg…]()

**Comment by saif-k** [comment]
![Screenshot_2022-07-13-21-31-03-77_577bcc70c040536a6d306ef99c704547](https://user-images.githubusercontent.com/38831824/178795215-ffd3606d-2f61-402e-934e-f2254f8c40db.jpg)

**Comment by reconman** [comment]
I have no idea how you came up with your offset values.

I see left=164, right=68, top=0, bottom=0

**Comment by saif-k** [comment]
Oh I just played around with the numbers, I'll test these values

**Comment by saif-k** [comment]
Yes thanks that solved the entire problem.

**Comment by reconman** [comment]
After I've updated to C.61 on my OnePlus 9, I also now have the same black bars.

I'll check and see if I can find any better solution than Custom Game Area mode.

**Comment by saif-k** [comment]
I just hope they don't keep changing the screen size.

**Comment by reconman** [comment]
On my phone I was able to get rid of the black bars by switching navigation mode to 3 buttons.

FGO doesn't like gesture navigation anymore for some reason. Because the black bars not only appear on OxygenOS, but also on LineageOS.

Can you test it on your device?

**Comment by saif-k** [comment]
Okay so I switched from gesture to buttons and the black bars to the right was removed but from the left side no since the camera is there , iv added a screenshot
So I also tested it on default for the game area mode with this change and it messes up during skill selection. Tested it with the working custom game area mode it doesn't run.
Reduced the right to 0 in the game area mode and it runs but it misses the first commands, I'm assuming it didn't recognize the skill location but it hits the 3rd skill correctly.
![Screenshot_2022-07-18-10-03-03-74_577bcc70c040536a6d306ef99c704547](https://user-images.githubusercontent.com/38831824/179453236-c7de424c-afb1-4173-8619-657199be67f5.jpg)
